### PR TITLE
Proper Return For Customer API

### PR DESCRIPTION
**Pull request by danielKNS**
This is a Proper Return for my Customer API

### CODE REVIEW

File: java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public String createNewCustomer(@RequestBody CustomerRequestBody customerRequest
     if(rows > 0){
       log.info("A new customer was CREATED!!!(has been inserted)");
     }
-    return sql;
+    return "Welcome "+ customerRequestBody.getFirstName() + "" + customerRequestBody.getLastName();
```
Comment (lgc13) [inline java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java:31]: hey Daniel. We talked about this last week. RestFul APIs should not be returning a string. Look back at what we said last week, as to what this API should return. Seems like you weren't paying attention...

**Review by lgc13 — CHANGES_REQUESTED** [review]
hey Daniel. We talked about this last week. RestFul APIs should not be returning a string. Look back at what we said last week, as to what this API should return. Seems like you weren't paying attention...

File: java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java [hunk 2]
```diff
@@ -21,18 +21,26 @@
...
-      log.info("A new customer was CREATED!!!(has been inserted)");
+      log.info("A new customer was CREATED!!!(has been inserted)" + newCustomer );
     }
-    return sql;
-    // the return goes straight to postman
-    // then goes to customerController line 41
+    return newCustomer;
+//     the return goes straight to postman
+//     then goes to customerController line 41
   }
```
Comment (LucasCostaAtCyberSaint) [inline java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java]: hey so these are not the changes we had agreed on yesterday

Comment (danielKNS) [reply java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java]: Let me fix that

Comment (danielKNS) [reply java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java]: Done. Is this the changes that  we agreed on yesterday ?

**Review by LucasCostaAtCyberSaint — APPROVED** [review]

File: java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java (hunk 2 above)
Comment (LucasCostaAtCyberSaint) [reply java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java]: yep

**Pull request merged**